### ISSUE TITLE
replace backbone but the loss cannot converge

### ISSUE THREAD

**Issue by githubgsq** [facebookresearch/maskrcnn-benchmark#577]
## ❓ Questions and Help
hi, all, I just replaced the backbone net in faster-rcnn, but the loss cannot converge and went up and down. The final AP is just 0.0003. Any advice please?

**Comment by LeviViana** [comment]
It's hard to give you an advice with no further information about your changes. Here are some general tips:

1. Did you set the parameters of your new backbone to `requires_grad = True` ?
2. In the e2e setup, the first conv group is not trained, i. e. `requires_grad = False`, it gives better stability to the training process
3. Did you trained for long enough ? Some times it takes tens of hours of training before noticing an improvement for deep architectures.
4. Are you adjusting your learning rates according to the batch size and the number of iterations already made ?
5. Are you using any type of normalization ? BatchNorm ? GroupNorm ? Any preprocessing transform ?

I hope these questions might help.

**Comment by githubgsq** [comment]
thanks @LeviViana , I will try it.

**Comment by fmassa** [comment]
Closing following @LeviViana comments. Thanks Levi!

**Comment by hunaodaotianming** [comment]
Hi, I have the same problem. Did you solve your problem?
